.circleci/tag_and_push_docker_image

```diff
@@ -9,7 +9,7 @@ function tag_and_push() {
   echo
   echo "Tagging and pushing $tag..."
   docker tag $built_tag $tag
-  docker push $tag
+  docker push $tag || docker push $tag || docker push $tag
```

Comment:
Chaining the command three times was a hack because the `dsd.io` registry stopped being reasonable. Do we still need it?